tests/test.h

```diff
@@ -149,4 +149,12 @@ void same_param_value(char* dev_name, uint64_t uid, char* param_name, param_type
  */
 void check_param_range(char* dev_name, uint64_t uid, char* param_name, param_type_t param_type, param_val_t expected_low, param_val_t expected_high);
 
+/**
+* Checks the latency between an action and the TIMESTAMP parameter on a TimeTestDevice
+ * Arguments:
+ *    uid: uniquie identifier of device
+ *    upper_bound_latency: The expected upperbound latency
+ *    start_time: The time right before the button is pressed
```

Comment:
Should specify that the `uid` is specifically for a `TimeTestDevice` (ik it's repetitive but it makes it clearer). Also typo :P 

`start_time` Mentions a button, though in the future it may not be a button press, so the function in general calculates the time difference between a `TimeTestDevice`'s `TIMESTAMP` and the provided "start" timestamp, which can be anything!